.travis.yml

```diff
@@ -14,6 +14,16 @@ env:
   - DISTRO=latest
 
 script:
+  # Run shellcheck on branch
+  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+  - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$PR, BRANCH=$BRANCH"
```

Comment:
Does't seem to pick up `$PR`

>$ echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$PR, BRANCH=$BRANCH"
TRAVIS_BRANCH=develop, PR=, BRANCH=script-cleanup